sw/device/lib/testing/test_framework/ottf.c

```diff
@@ -35,10 +35,8 @@ static void init_uart(void) {
   base_uart_stdout(&uart0);
 }
 
-static void freertos_test_task(void *task_parameters) {
-  bool result = test_main();
-
-  // Must happen before any debug output.
+static void report_test_status(bool result) {
+  // Reinitialize UART before print any debug output if the test clobbered it.
   if (kTestConfig.can_clobber_uart) {
     init_uart();
   }
```

Comment:
This does not apply to DV, right?

Reply:
Oh good catch! This was lingering from the original test_main.c.

Fixed.

Reply:
Actually the condition on line 40 should also not apply to DV.

Reply:
ah right, my bad. Fixed.